routes/index.js

```diff
@@ -15,9 +15,26 @@ router.get('/', function(req, res) {
 
 /* GET /notebooks - list of notebooks */
 router.get('/notebooks', function(req, res) {
-    res.render('index', {
-        title: 'Notebooks'
-    });
+    nbstore.getNotebooks().then(
+        function success(notebooks) {
+            //if notebook index exists redirect to it immediately
+            var notebookIndex = "index.ipynb"
+            if(notebooks.indexOf(notebookIndex) > -1) {
+                res.redirect('/notebooks/' + notebookIndex);
+            }
```

Comment:
This only looks for "index" name, but not "Index" or "INDEX". Is that fine, or should we make this case-insensitive? I lean towards the latter.